Xamarin.Forms.Core/WebView.cs

```diff
@@ -145,5 +179,39 @@ public void SendNavigating(WebNavigatingEventArgs args)
 		{
 			return _platformConfigurationRegistry.Value.On<T>();
 		}
+
+		private static string EscapeJsString(string js)
+		{
+			if (!js.Contains("'"))
```

Comment:
Because we add an extra eval function in the code I added for standardizing platforms, we need to add an extra level of escaping to any single quotes the user may have passed in for their script. I really don't like this but it was all I could come up with to solve this problem. I'm open to suggestions here.